components/constellation/pipeline.rs

```diff
@@ -119,7 +119,7 @@ pub struct InitialPipelineState {
     pub layout_to_constellation_chan: IpcSender<LayoutMsg>,
 
     /// A channel to schedule timer events.
-    pub scheduler_chan: IpcSender<TimerSchedulerMsg>,
+    pub scheduler_chan: Sender<TimerSchedulerMsg>,
```

Comment:
It does, but I think they were working off of the instructions here: https://github.com/servo/servo/issues/15219#issuecomment-275684028

```pub devtools_chan: Option<Sender<DevtoolsControlMsg>>``` a couple of lines down also violates this rule FWIW, would this case be similarly justified?

Reply:
Ok, it looks to me like that comment is out of date and should be removed. InitialPipelineState can contain Senders; it's UnprivilegedPipelineContent that cannot, and that is now enforced by the compiler because of the Serialize implementation that is derived for it.